config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  resources :users, only: %i[new create]
```

Comment:
- [ ] Currently, I am unable to edit a user. To be able to edit a user, the route for edit should also be added.e.g.
```ruby
resources :users, only: [:new, :create, :edit, :update]
```